install.sh

```diff
@@ -304,8 +284,7 @@ Usage: "${0}" [options]
 Options:
 
 --develop       code remains editable in place (currently always enabled)
---python, -p    python version (default: python3)
```

Comment:
I'm against removing `--python` option from `install.sh`. First, I see no need to force complete removal of Python2 for now, it's enough to remove tests and add note to docs about Python2 not being supported anymore. Secondly, you can use it to specify subversion of Python3 in case multiple versions are installed. This was once an issue for me on a specific computer, see #259 and #262

Reply:
IMO once tests are disabled for python2 we really should completely prevent joinmarket from running under python2 at all as there could be subtle incompatibilities.

Reply:
@kristapsk that was an oversight. I agree that it should be possible to use different python3 versions and I've restored the option displaying an error and halting when any version other than python3 is specified.